Sources/ApolloAPI/GraphQLOperation.swift

```diff
@@ -6,33 +6,92 @@ public enum GraphQLOperationType {
   case subscription
 }
 
+/// The means of providing the operation document that includes the definition of the operation
+/// over network transport.
+///
+/// This data represents the `Document` as defined in the GraphQL Spec.
+/// - See: [GraphQLSpec - Document](https://spec.graphql.org/draft/#Document)
+///
+/// The Apollo Code Generation Engine will generate the `DocumentType` on each generated
+/// `GraphQLOperation`. You can change the type of `DocumentType` generated in your
+/// [code generation configuration](// TODO: ADD URL TO DOCUMENTATION HERE).
```

Comment:
should this be updated to a warning-todo, or are you going to fix this before the PR is merged?

Reply:
I'll fix this before releasing 1.0. My release 1.0 check-list includes running a find for "TODO" and cleaning them all up :)

Reply:
I use warning todos a lot too, but I won't be able to fix this one until after we have documentation for the Code Gen Engine itself, so that will be a while.